integration_tests/elastic_search_test.go

```diff
@@ -73,6 +73,7 @@ var _ = Describe("Broker Compose Integration", func() {
 			putURI := binding.Credentials.URI + "twitter/tweet/1?op_type=create"
 			putData := "{\"user\" : \"kimchy\",\"post_date\" : \"2009-11-15T14:12:12\",\"message\" : \"trying out Elasticsearch\"}"
 			request, err := http.NewRequest("PUT", putURI, strings.NewReader(putData))
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
you can thank [gometalinter](https://github.com/alecthomas/gometalinter) for that